plotting/private/bg_rgba2rgb.m

```diff
@@ -0,0 +1,116 @@
+function [rgb] = bg_rgba2rgb(rgba, bg, varargin)
```

Comment:
I suggest to make the order of the input arguments consistent with the name of the function.